automation/jenkins-package-feed.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+
+set -e
+
+BUILD_CONTAINER_NAME=yocto-build-$$
+
+print_help() {
+	echo -e "Script options:\n\
+	\t\t -h | --help\n
+	\t\t -m | --machine\n\
+	\t\t\t (mandatory) Machine to build for. This is a mandatory argument\n
+	\t\t --shared-dir\n\
+	\t\t\t (mandatory) Directory where to store shared downloads and shared sstate.\n
+	\t\t -b | --build-flavor\n\
```

Comment:
All references to build flavor should be removed anyway since we'd only build for prod, right? Or would we need also to build .dev variants of the packages?